po/pt_BR.po

```diff
@@ -98,15 +102,20 @@ msgid ""
 "mode."
 msgstr "Seguir as cores do sistema ou forçar o modo claro ou escuro."
 
-#: src/exm-application.c:111
+#. TRANSLATORS: 'Name <email@domain.com>' or 'Name https://website.example'
+#: src/exm-application.c:112
+msgid "translator-credits"
+msgstr "Tiago Lucas Flach https://github.com/tiagoFlach"
```

Comment:
As @douglasilvadev contributed a large portion of these translations, let's credit him here as well.
```suggestion
msgstr ""
"Douglas Silva <douglasilvadev@gmail.com>"
"Tiago Lucas Flach https://github.com/tiagoFlach"
```